plugins/samson_airbrake/config/initializers/airbrake.rb

```diff
@@ -4,11 +4,13 @@
   Airbrake.user_information = # replaces <!-- AIRBRAKE ERROR --> on 500 pages
```

Comment:
pull user_information_placeholder abvoe and change to 
`replaces user_information_placeholder `